tests/qos/test_qos_sai.py

```diff
@@ -1118,7 +1118,7 @@ def testQosSaiLossyQueue(
     def testQosSaiLossyQueueVoq(
         self, LossyVoq, ptfhost, dutTestParams, dutConfig, dutQosConfig,
             ingressLossyProfile, duthost, localhost, get_src_dst_asic_and_duts,
-            skip_src_dst_different_asic
+            skip_src_dst_different_asic,  separated_dscp_to_tc_map_on_uplink
```

Comment:
@rraghav-cisco 
conflict on 202305 branch,
can you file a separate PR to doble commit for 202305 branch?

Reply:
@XuChen-MSFT : 3 PRs are open:
202205: https://github.com/sonic-net/sonic-mgmt/pull/10381
master: https://github.com/sonic-net/sonic-mgmt/pull/10896
202305: https://github.com/sonic-net/sonic-mgmt/pull/10898